infra/partner-vm-testnet/resources.tf

```diff
@@ -1,7 +1,7 @@
 terraform {
   backend "gcs" {
-    bucket = "multichain-terraform-dev" # Example: terraform-multichain-state-bucket
-    prefix = "state/multichain-vm-partner-test" # Example: state/multichain-vm
+    bucket = "multichain-terraform-dev"
```

Comment:
backend "gcs" {
    bucket = "multichain-terraform-{your_entity_name}"
    prefix = "state/testnet"
  }
This is what they will use as in the doc. Let's make it consistent :)